packages/components-angular/tests/e2e/specs/tabs-bar-navigation-bug.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import type { Page } from 'puppeteer';
+import { goto } from '../helpers';
+
+let page: Page;
+beforeEach(async () => (page = await browser.newPage()));
+afterEach(async () => await page.close());
+
+it('should correctly set activeTabIndex on p-tabs-bar', async () => {
+  await goto(page, 'tabs-bar-navigation-bug');
+
+  const getTabsBarActiveTabIndex = (): Promise<number> => page.$eval('p-tabs-bar', (el) => (el as any).activeTabIndex);
```

Comment:
```suggestion
  const getTabsBarActiveTabIndex = (): Promise<number> => page.$eval('p-tabs-bar', (el) => (el as PTabsBar).activeTabIndex);
```

Reply:
While this contains the key `activeTabIndex`, it is the wrong type since the angular class is different from the custom html element that we want to type here.

<img width="555" alt="Bildschirmfoto 2023-11-22 um 08 58 44" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/ed0ef76c-b815-4642-bafe-fc65beadbc37">